seemabansal2003.md

```diff
@@ -0,0 +1,55 @@
+---
+name: "@seemabansal2003"
+project: "TechnoFlora"
+---
+
+# TechnoFlora
+
+## Summary
+
+I come from a family of gardening enthusiasts who love to spend time with their plants. However whenever we go on a vacation, most of our plants die due to lack of care. Thus, being an Arduino Enthusiast, i want to build an automatic hydroponics based plant incubator called TechnoFlora. TechnoFlora would feature technologies like hydroponics, grow lights & temperature and humidity control that would not only help the plant in growing during adverse conditions but also faster than normal during better times. I can't wait to grow and completely harvest lettuce within just a couple of weeks!
+
+## Plan
+
+First of all, TechnoFlora would feature a hydroponics tray with water level indicators that can act as the medium of plant growth, nourishing the plant with the required minerals. Then it would also feature a full spectrum grow light which can artificially stimulate photosynthesis and help plants grow faster. Along with this, It would be covered with an acrylic enclosure that would not only trap stray light but also help in making a controlled environnment around the plant. There would be a temperature & humidity control system featuring a DHT11 sensor & vntilation and cooling fans to control the plants surroundings. All information regarding the plant's growth would be synced with a mobile application using bluetooth. This would not only help me monitor the plant better but also help it grow faster.
+
+## Budget
+
+
+| Product                                                | Supplier/Link                     | Cost   |
+| --------------------------------------------------------------------- | ---------------------------------------------- | ---------- |
+|hydroponics tray |https://amzn.eu/d/4M5MgpP| $43.50 |
+|Hydroponics Nutrient |https://amzn.eu/d/f0NNvWq| $5.14 |
+| TDS Meter |https://amzn.eu/d/dWoPVoC| $3.61 |
```

Comment:
how would this be integrated into your system? is there a way for you to read data from this meter through your arduino instead of just their screen?

Reply:
> how would this be integrated into your system? is there a way for you to read data from this meter through your arduino instead of just their screen?

I do plan to integrate the sensor present in the meter with the arduino. However I will need to read the analog voltage signals provided by the sensor at known TDS levels to do so.

Reply:
ahhh ok that makes sense, thanks!